packages/aws-cdk-lib/aws-ecs-patterns/README.md

```diff
@@ -70,6 +70,10 @@ By setting `redirectHTTP` to true, CDK will automatically create a listener on p
 
 If you specify the option `recordType` you can decide if you want the construct to use CNAME or Route53-Aliases as record sets.
 
+If you specify the option `containerCpu` allows you to set the minimum number of CPU units to reserve for the container.
+
+If you specify the option `containerMemoryLimitMiB` allows you to set the amount of memory (in MiB) to provide to the container.
+
```

Comment:
Could you add the properties to the sample code above?

```ts
declare const cluster: ecs.Cluster;
const loadBalancedFargateService = new ecsPatterns.ApplicationLoadBalancedFargateService(this, 'Service', {
  cluster,
  memoryLimitMiB: 1024,
  cpu: 512,
  taskImageOptions: {
    // ...
```

Reply:
For no apparent reason, I set the value of each property to half the value of 'cpu','memoryLimitMiB'. Is there a problem?

https://github.com/aws/aws-cdk/pull/30920/commits/3f2f3daf007ec28077913a9ca3f76b20b9b09480

Reply:
No problem.